src/core/qgsfields.h

```diff
@@ -142,10 +144,10 @@ class CORE_EXPORT  QgsFields
     void extend( const QgsFields &other );
 
     //! Checks whether the container is empty
-    bool isEmpty() const;
+    Q_INVOKABLE bool isEmpty() const;
```

Comment:
Did you consider switching these getters to `Q_PROPERTY`?

Reply:
@m-kuhn , I hadn't, but yeah, it could well be that, and would be more QML-esque :)